### DISCUSSION TITLE
Welcome to spicetify-bloom Discussions!

### DISCUSSION THREAD

**Discussion by nimsandu** [nimsandu/spicetify-bloom#86] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ayakamomo** [comment]
the sidebar playlists to show is also error，this guide seems unable to improve this mistake
![QQ截图20230506005438](https://user-images.githubusercontent.com/132696281/236519716-369c93e0-d675-4409-9b22-76677a80ae1d.png)

**Reply by SunsetTechuila** [reply]
@Ayakamomo Hello! It is [known](https://github.com/nimsandu/spicetify-bloom/issues/153) issue and I'm [working](https://github.com/nimsandu/spicetify-bloom/tree/libraryX) on it. [This](https://github.com/nimsandu/spicetify-bloom/issues/153#issuecomment-1503768653) is a temporally solution.
Please use issues and not discussions to report about problems with the theme.
